Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -36,8 +36,11 @@ public static class NugetHelper
         /// <summary>
         /// The path where to put created (packed) and downloaded (not installed yet) .nupkg files.
         /// </summary>
+#if UNITY_EDITOR_OSX
+        public static readonly string PackOutputDirectory = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData) + "/NuGet/Cache";
```

Comment:
Would `Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData),"NuGet","Cache")` work on all platforms?

Reply:
Yes, but 2 Path.Combine invokes would be used

Reply:
That seems ok right?

Reply:
Yes, only 2 arguments needed

Reply:
Thanks for the change! =)